Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,32 @@
+# rkt stop
+
+Given a list of pod UUIDs, rkt stop will shut them down, for the shipped stage1 images, this means:
+
+* default systemd-nspawn stage1: the apps in the pod receive a TERM signal and, after a timeout, a KILL signal.
+* kvm stage1: the virtual machine is shut down with `systemctl halt`.
+* rkt fly stage1: the app receives a TERM signal.
+
+The `--force` flag will stop a pod forcibly, that is:
+
+* default systemd-nspawn stage1: the container is killed.
```

Comment:
@alban You're right, and that was the problem since I implemented recursive killing of child processes. The question is whether this makes any sense or should we limit only to sending SIGTERM.